spec/koans.js

```diff
@@ -270,13 +272,13 @@ describe("the JavaScript language", function() {
       var z = true;
 
       example(x);
-      //expect(x).toEqual();
+      expect(x).toEqual("test!"); //what??
```

Comment:
When we pass a value to a function, we pass a copy, so outside the function, the variables keep their value.
When we say: arg = "test" we only change the value of arg (that is a copy of x, y or z)